src/components/include/application_manager/application_manager.h

```diff
@@ -873,18 +873,24 @@ class ApplicationManager {
                                protocol_handler::ServiceType service_type) = 0;
 
   /**
-   * @brief Called when application completes streaming configuration
+   * @brief Called when application successfully completes streaming
+   * configuration
    * @param app_id Streaming application id
    * @param service_type Streaming service type
-   * @param result true if configuration is successful, false otherwise
-   * @param rejected_params list of rejected parameters' name. Valid
-   *                        only when result is false.
    */
-  virtual void OnStreamingConfigured(
+  virtual void OnStreamingSuccessfulConfiguration(
+      uint32_t app_id, protocol_handler::ServiceType service_type) = 0;
+
+  /**
+   * @brief Called when application fails streaming configuration
+   * @param app_id Streaming application id
+   * @param rejected_params list of rejected parameters' name
```

Comment:
```suggestion
   * @param rejected_params list of rejected parameter names
```

Reply:
Done in 1c6840d4d5edcef26befb640c2dcc57a9c2fa20c